src/Templates/NServiceBusWindowsService/Program.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+using System.ServiceProcess;
+using System.Threading.Tasks;
+
+namespace NServiceBusWindowsService
+{
+    internal static class Program
+    {
+        public async static Task Main(string[] args)
```

Comment:
It's a good point, and I'm glad you brought it up. I committed this with some doubt and I was hoping that someone would call it out. 😄 

Perhaps C# 7.1 is too much for now. What would we find acceptable? 7.0? 6.0? Interestingly, while we should ensure that the templates comply with the minimum version we want to support, the content of the `langVersion` tag in the templates is another matter. E.g. it could still say `latest`.

Reply:
Given that we are using SDK-style csproj, the oldest version of VS supported is 2017, and that comes with C# 7.0, so I propose that we change language version to C# 7.0.